omero/sysadmins/repository-move.txt

```diff
@@ -0,0 +1,100 @@
+Moving the data repository
+==========================
+
+It may be necessary to move either the whole OMERO data directory or only the
+Managed Repository to a new location on the file system. This should be done
+with care following the steps below.
+
+.. warning::
+    Before moving OMERO data it is wise to ensure that both the data and the
+    database are fully backed up. See :doc:`server-backup-and-restore`.
+
+The current location of the data repositories can be found using the
+:omerocmd:`fs repos` command:
+
+::
+
+    $ bin/omero fs repos
+
+     # | Id | UUID                                 | Type    | Path
+    ---+----+--------------------------------------+---------+-----------------------------------------
+     0 | 1  | 309ea513-a23c-48d1-abd2-9ceed1b3ffa4 | Managed | /Users/omero/var/omero/ManagedRepository
+     1 | 2  | ScriptRepo                           | Script  | /Users/omero/dist/lib/scripts
+     2 | 3  | 3ec8c878-c776-48a3-b57e-2a11b0c97045 | Public  | /Users/omero/var/omero
+    (3 rows)
+
+Moving the OMERO data directory
+-------------------------------
+
+If the Managed Repository is within the OMERO data directory and the whole
+data directory is to be moved then the following steps should be used:
+
+::
+
+    bin/omero config omero.data.dir NEW
```

Comment:
Thanks @joshmoore. That'll teach me to cut and paste from my own CLI rather than the ticket https://trac.openmicroscopy.org/ome/ticket/12355#comment:5 comment!